core/chains/evm/client/rpc_client.go

```diff
@@ -376,6 +377,10 @@ func (r *RPCClient) BatchCallContext(rootCtx context.Context, b []rpc.BatchElem)
 	var requestedFinalizedBlock bool
 	if r.chainType == chaintype.ChainAstar {
 		for _, el := range b {
+			if el.Method == "eth_getLogs" {
+				r.rpcLog.Critical("evmclient.BatchCallContext: eth_getLogs is not supported")
+				return errors.New("evmclient.BatchCallContext: eth_getLogs is not supported")
+			}
```

Comment:
Does this mean we can't support LogPoller / CCIP on AStar?

Reply:
Oh, nevermind... I forgot we only use batch requests with `eth_getBlockByNumber`.  Still, very strange that AStar wouldn't support this! We have thought about changing the `eth_getLogs` requests to a batched request, I guess if we do that we'll have to fall back to the older method on AStar 🤔